examples/python/Misc/MultiDevice/multi_device_frame_sync.py

```
@@ -245,6 +260,28 @@ def interruptHandler(sig, frame):
+decoders = {}
+lastDecodedFrames = {}
+
+def decodeH265(outputName, msg):
+    decoder = decoders.get(outputName)
+    if decoder is None:
+        decoder = av.CodecContext.create("hevc", "r")
+        decoders[outputName] = decoder
+
+    try:
+        frames = decoder.decode(av.Packet(bytes(msg.getData())))
+    except Exception:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)